content/en/user-guide/web-application/resource-browser/index.md

```diff
@@ -5,60 +5,70 @@ description: >
 
 <img src="resource-browser.png" alt="LocalStack Web Application's Resource Browsers outlining various local AWS services" title="Resource Browser" width="900" />
 
-{{< alert title="Warning" color="warning">}}
-- If you encounter a `Network Failure` error message while accessing the Resource Browser, it is likely that the LocalStack container is not running or the instance is not reachable at the endpoint specified in the instance bookmark.
-{{< /alert >}}
-
-{{<alert title="Note">}}
-- An AWS region dropdown menu in the dashboard is located on the top right of the page. You can select your desired region to ensure that you can view your resources. If you cannot view resources that you have recently created, you should verify that you are checking the resources in the correct region.
+The Resource Browser provide an experience similar to the AWS Management Console. However, the Resource Browser is not a replacement for the AWS Management Console and only replicate some of the features of the AWS Management Console. We recommend using our [integrations](https://docs.localstack.cloud/user-guide/integrations/) to create your resources, with the Resource Browser being used for quick viewing and management of your resources.
 
-- The Resource Browser provide an experience similar to the AWS Management Console. However, the Resource Browser is not a replacement for the AWS Management Console and only replicate some of the features of the AWS Management Console. We recommend using our [integrations](https://docs.localstack.cloud/user-guide/integrations/) to create your resources, with the Resource Browser being used for quick viewing and management of your resources.
+The LocalStack Web Application connects to your LocalStack container and retrieves the information about your local resources directly via the `localhost` without using the internet. None of the information is sent to the internet, or stored on any external servers maintained by LocalStack.
 
-- To view the running services in your LocalStack container, you can use the [System Status](https://app.localstack.cloud/status). This view provides a quick overview of the running services in your LocalStack container. To get a detailed telemetry of your API calls to your LocalStack containers, check out [Stack Insights](https://docs.localstack.cloud/user-guide/web-application/stack-insights/).
+{{<alert title="Switching AWS regions">}}
+An AWS region dropdown menu in the dashboard is located on the top right of the page. You can select your desired region to ensure that you can view your resources. If you cannot view resources that you have recently created, you should verify that you are checking the resources in the correct region.
 {{< /alert >}}
 
 ## Supported services
 
-We currently support the following local AWS services in our Resource Browser:
+The Resource Browser supports the following AWS services:
```

Comment:
Should we make some sort of auto-script that mutates this table automatically when a new RB is added?

Reply:
Yes, but let's keep it as a future item 👍